### PR TITLE
docs: add custom styling & custom logo

### PR DESCRIPTION
**Pull request by thim81**
**Description**

Changes proposed in this pull request:

- add documentation on how to apply custom CSS styling
- add documentation on how to show a custom logo

**Related issue(s)**
#530

### CODE REVIEW

**Comment by magicmatatjahu** [comment]
@thim81 Hi! I will review tomorrow, don't worry :)

File: README.md
Comment (Samridhi-98) [inline README.md:164]:
```suggestion
If you are using the standalone bundle, you can put the custom font styling as a style sheet link or as an inline style in the `<head>` section of the HTML code.
```
Please correct the spelling. 😊

**Comment by thim81** [comment]
If possible do a "Squash and merge" since the spelling & indenting corrections are too much :-)

File: README.md
Comment (magicmatatjahu) [inline README.md:140]:
```suggestion
The AsyncAPI React component does not set any global fonts and any global reset styles. This provides the option to define your custom `font-family` and other styling.

This can be done by defining the styles in a file or inline using by adding a `<style>` tag in the `<head>` section of the page where you are using AsyncAPI React component.
```

File: README.md
Comment (magicmatatjahu) [inline README.md:142]:
```suggestion
Example custom styles (used in the [AsyncAPI studio](https://github.com/asyncapi/studio)):

```

File: README.md
Comment (magicmatatjahu) [inline README.md:162]:
```suggestion
If you are using the component in project project that uses bundler like Webpack don't forget to import the custom styles.

``` js
import "styles/custom.css";
import "@asyncapi/react-component/styles/default.min.css";
```
```

Custom styles should be applied first. Css don't override styles but uses first defined. :)

File: README.md
Comment (magicmatatjahu) [inline README.md:202]: I think that here we should only focus on the style definition, so please change it to the:

 ```suggestion
If you are using the [standalone bundle](./docs/usage/standalone-bundle.md), you can put the custom styles as a style sheet link or as an inline style in the `<head>` section of the HTML code:

```html
  <head>
    <!-- Custom style sheet -->
    <link rel="stylesheet" href="./styles/custom.css">

    <!-- OR as inline style -->
    <style>
      html{-moz-tab-size:4;-o-tab-size:4;tab-size:4;line-height:1.15;-webkit-text-size-adjust:100%};
      body{margin:0;font-family:system-ui,-apple-system,Segoe UI,Roboto,Helvetica,Arial,sans-serif,Apple Color Emoji,Segoe UI Emoji};
    </style>
    
    <link rel="stylesheet" href="https://unpkg.com/@asyncapi/react-component@1.0.0-next.32/styles/default.min.css">
    
    ...
  </head>
```
```

File: README.md
Comment (magicmatatjahu) [inline README.md:206]:
```suggestion
The AsyncAPI component supports the option to use a custom logo. By using the `x-logo` custom extension in the [InfoObject](https://github.com/asyncapi/spec/blob/master/spec/asyncapi.md#infoObject), a logo will be shown in the top left corner.

> **NOTE**: The logo will only appear if the [sidebar option](./docs/configuration/config-modification.md#definition) is enabled.
```

File: README.md
Comment (magicmatatjahu) [inline README.md:213]:
```suggestion
  description: This service is in charge of processing user signups.
```

File: README.md
Comment (magicmatatjahu) [inline README.md:220]: We shouldn't show any picture :) Please remove it.

```suggestion
```

**Review by magicmatatjahu — CHANGES_REQUESTED** [review]
@thim81 Thanks for awesome contribution! However I have 7 suggestions how we should improve new docs :) Please refer to them.

**Comment by magicmatatjahu** [comment]
@thim81 

> If possible do a "Squash and merge" since the spelling & indenting corrections are too much :-)

Don't worry, we always squash before merge, we have enabled it on the Github side :)

File: README.md
Comment (thim81) [reply README.md:143]: I refer to the "styles/custom.css" a bit later, so perhaps we can keep it?

Comment (magicmatatjahu) [reply README.md:143]: I thought the user would understand that this name refers to the custom styles, but you could add information that the file is called `styles/custom.css` like:

```md
Example custom styles (defined in the `styles/custom.css` file):
```

Also we don't need to add info where that style is used.

Comment (thim81) [reply README.md]: Agreed, I had included the "AsyncAPI studio" style since that was the style that is used in the custom example.

I have updated the readme and removed the AsyncAPI studio part.

**Review by magicmatatjahu — COMMENTED** [review]
@thim81 Hi! Please ad in some places missed "`" chars because I see that we have broken md file. Github suggestion system has problem with "`"...

**Comment by thim81** [comment]
@magicmatatjahu The missing quotes are back and the page should look OK again.

File: README.md
Comment (magicmatatjahu) [inline README.md:141]:
```suggestion
This can be done by defining the styles in a file or inline using a `<style>` tag in the `<head>` section of the page where you are using AsyncAPI React component.
```

**Review by magicmatatjahu — COMMENTED** [review]
Ok, last suggestion and for me everything looks good! @alequetzalli Do you wanna also make review?

**Review by magicmatatjahu — APPROVED** [review]
LGTM! We should wait for response from @alequetzalli  )

File: README.md
Comment (alequetzalli) [inline README.md:139]:
```suggestion
### Custom styles
The AsyncAPI React component does not set any global fonts or reset styles. This component provides the option to define your custom `font-family` and other styling.
```

File: README.md
Comment (alequetzalli) [inline README.md:159]:
```suggestion
If you are using the component in a project that uses a bundler like Webpack, don't forget to import the custom styles.
```

**Review by alequetzalli — COMMENTED** [review]
Left my review! ✨❤️ Just a few suggestions. 

@magicmatatjahu @thim81

File: README.md
Comment (thim81) [reply README.md:139]: `The AsyncAPI React component does not set any global fonts and any global reset styles`
The "or" would not be sufficient since it there are no global fonts sets and a number of styles are reset.

Comment (thim81) [reply README.md]: I updated that part.

**Comment by magicmatatjahu** [comment]
@alequetzalli Thanks for review! :)

File: README.md
Comment (magicmatatjahu) [inline README.md:139]: @thim81 The component doesn't reset `a number of styles` 😅

**Review by magicmatatjahu — CHANGES_REQUESTED** [review]

File: README.md
Comment (thim81) [reply README.md:139]: I ll take it out 🤦‍♂️

**Review by alequetzalli — APPROVED** [review]
🚀  LGTM

**Comment by thim81** [comment]
@magicmatatjahu According to Github there is a still a change request open? But from what I can see all the comments have been resolved? Can you tell me if you were missing something?

**Comment by magicmatatjahu** [comment]
@thim81 Hi! I felt bad on Thursday and Friday and didn't work. Sorry for the delay. Everything is ok. Thanks for your contribution! ❤️

**Review by magicmatatjahu — APPROVED** [review]
LGTM!

**Pull request merged**
